dashboard/config/scripts_json/allthethings.script_json

```diff
@@ -5254,6 +5254,27 @@
     },
     {
       "chapter": 133,
+      "position": 3,
+      "activity_section_position": 3,
```

Comment:
Just wondering if we should create a new lesson for Dance Party - AI instead of adding a level to legacy Dance Party. Then the lesson will be closer to the other AI-related lessons such as AI Chat and AI Rubrics as a comparison.

Reply:
Since preview will not be added to legacy for now, these new levels would be where preview is set to true.

Reply:
We should curate `allthethings` so that each lab has a single lesson as much as possible.  Given that this change is a small tweak to existing functionality, I think a new level added to the existing lesson will be best.

Reply:
That said, I think that potential merge conflicts in this file would be challenging to deal with, so I'm not going to modify this file in this PR after all, and will instead add the new level on the levelbuilder machine when this PR makes it there.

Reply:
Sounds good - thanks!